src/main/java/org/dependencytrack/policy/PolicyEngine.java

```diff
@@ -132,6 +133,7 @@ private PolicyViolation.Type determineViolationType(final PolicyCondition.Subjec
             case CPE:
             case SWID_TAGID:
             case COMPONENT_HASH:
+            case VULNERABILITY_ID:
```

Comment:
VULNERABILITY_ID will come under SECURITY violation type. So, may be move it before `return PolicyViolation.Type.SECURITY;`